src/User/Persistence.hs

```diff
@@ -9,4 +10,7 @@ selectAllUsers :: (HasPostgres m) => m [User]
 selectAllUsers = query_ allUsersQuery
 
 insertUser :: (HasPostgres m) => UserForm -> m Int64
-insertUser = execute insertUserQuery
\ No newline at end of file
+insertUser = execute insertUserQuery
+
+deleteUser :: (HasPostgres m) => Int -> m Int64
+deleteUser key = execute deleteUserQuery $ toRow [key]
```

Comment:
@kinorius123 Is the `toRow` function necessary?

Reply:
@CristhianMotoche You are right, removing it right now.